### PR TITLE
Mortar Pestle Bonk initial upload

### PR DESCRIPTION
**Pull request by Ben-Colwell**
A simple plugin that changes the sounds of the mortar and pestle when crushing items to play a bonking sound instead.

### CODE REVIEW

File: plugins/mortar-pestle-bonk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Erishion-Games-LLC/Mortar_Pestle_Runelite_Plugin.git
+commit=a23f11010dae0935b06ec3e1828920ad078cbcad
```
Comment (Nightfirecat) [inline plugins/mortar-pestle-bonk:2]: I recommend adding an `authors` field to get rid of the bot's "non-author plugin change" label.
```suggestion
commit=a23f11010dae0935b06ec3e1828920ad078cbcad
authors=Ben-Colwell
```

**Comment by raiyni** [comment]
Remove `log.info` calls

**Comment by Ben-Colwell** [comment]
Sorry didn't mean to click that

**Comment by Nightfirecat** [comment]
[Don't use "example" as your config group](https://github.com/Erishion-Games-LLC/Meme-Sounds-Runelite-Plugin/blob/49113e555895ca3de6d7f56018cba91bab7636c9/src/main/java/com/erishiongames/mortarpestle/MortarPestlePluginConfig.java#L7)

**Comment by Nightfirecat** [comment]
I am closing this PR due to inactivity. You can feel free to reopen it if the issues mentioned above are resolved and we'll re-review it.

**Pull request closed without merging**